presto-common/src/main/java/com/facebook/presto/common/predicate/TupleDomainFilter.java

```diff
@@ -1327,6 +1446,14 @@ public String toString()
         private final BigintRange[] ranges;
         private final long[] longLowerBounds;
 
+        private static final int INSTANCE_SIZE = ClassLayout.parseClass(BigintMultiRange.class).instanceSize();
+
+        @Override
+        public long getRetainedSizeInBytes()
```

Comment:
These methods are out of order. Presto follows the following order
1. public with @Override
2. public
3. protected
4. package default
5. private
It might be good to create another commit to reorder these methods for all classes